supermarq-benchmarks/supermarq/qcvv/irb.py

```diff
@@ -405,9 +414,19 @@ def average_error_per_clifford_std(self) -> float:
             raise self._not_analyzed
         return self._average_error_per_clifford_std
 
-    def plot_results(self) -> None:
-        """Plot the exponential decay of the circuit fidelity with cycle depth."""
+    def plot_results(
+        self,
+        filename: str | None = None,
+    ) -> None:
+        """Plot the exponential decay of the circuit fidelity with cycle depth.
+
+        Args:
+            filename: Optional argument providing a filename to save the plots to. Defaults to None,
+                indicating not to save the plot.
+        """
         super()._plot_results()
```

Comment:
predates this pr, but this should just be
```suggestion
        self._plot_results()
```
the only reason to use super() would be if `_plot_results()` was overridden in this subclass (and we wanted to use the original version)

(same occurs a few other places as well)

Reply:
Yeah not sure how that slipped through